test/suites/ACID.m

```diff
@@ -671,12 +673,20 @@
        exp(.05*x).*cos(x)]';
   h = stem(x, y);
   legend( 'exp(-.07x)*cos(x)', 'exp(.05*x)*cos(x)', 'Location', 'NorthWest');
-  set(h(1),'MarkerFaceColor','blue')
-  set(h(2),'MarkerFaceColor','red','Marker','square')
+  set(h(1),'MarkerFaceColor','blue');
+  set(h(2),'MarkerFaceColor','red','Marker','square');
+
+  % Octave 4 has some smart behavior: it only prints a single baseline.
+  % Let's mimick this behavior everywhere else.
+  baselines = findall(gca, 'Type', 'line', 'Color', [0 0 0]);
+  if numel(baselines) > 1
+      delete(baselines(1:end-1));
```

Comment:
I know. I did it that way the first time in e0476871ea15500ddecab6d577cc64bcacdab2e2, but then changed it to this code to exactly match the behavior between Octave 3.8 and 4.0.

The structure of the plot was something like:
 - data
 - baseline
 - data
 - baseline
in Octave 3.8 and I think
 - data
 - data
 - baseline
in Octave 4.0. So that's the reason for this. I will add a comment in the code to clarify.